sources/academy/platform/getting_started/creating_actors.md

```diff
@@ -1,131 +1,180 @@
 
-To make this simple Actor functional, let's delete the current value of `result`, and replace it with the numbers we are getting from the user's input:
+You will end up on a template detail page where you can see the all the important information about the template - description, included features, used technologies, and what is the use-case of this template. More importantly, there is code preview and also instructions for how the code works.
 
-```js
-const result = (input?.firstNumber || 0) + (input?.secondNumber || 0);
+![JavaScript template detail page](./images/create-actor-template-detail-page.png)
+
+### Using the template in the Web IDE {#web-ide}
+
+By clicking **Use this template** button you will create the Actor in Apify Console and you will be moved to the **Code** tab with the [Web IDE](/platform/actors/development/quick-start/web-ide) where you can see the code of the template and start editing it.
+
+> The Web IDE is a great tool for developing your Actor directly in Apify Console without the need to install or use any other software.
+
+![Web IDE](./images/create-actor-web-ide.png)
+
+### Using the template locally {#local}
+
+If you want to use the template locally, you can again use our [Apify CLI](/cli) to download the template to your local machine.
+
+> Creating an Actor from a template locally is a great option if you want to develop your code using your local environment and IDE and then push the final solution back to the Apify Console.
+
+When you click on the **Use locally** button, you'll be presented with instructions on how to create an Actor from this template in your local environment.
+
+With the Apify CLI installed, you can simply run following commands in your terminal:
+
+```bash
+apify create my-actor -t getting_started_node
+```
+
+```bash
+cd my-actor
+apify run
 ```
 
-The final code should look like this:
+![Use the template locally](./images/create-actor-template-locally.png)
+
+## Start with scraping single page {#scraping-single-page}
+
+This template is a great starting point for web scraping as it extracts data from a single website. It uses [Axios](https://axios-http.com/docs/intro) for downloading the page content and [Cheerio](https://cheerio.js.org/) for parsing the HTML from the content.
+
+Let's see what's inside the **Start with JavaScript** template. The main logic of the template lives in the `src/main.js` file.
 
 ```js
+// Axios - Promise based HTTP client for the browser and node.js (Read more at https://axios-http.com/docs/intro).
+import axios from 'axios';
+// Cheerio - The fast, flexible & elegant library for parsing and manipulating HTML and XML (Read more at https://cheerio.js.org/).
+import * as cheerio from 'cheerio';
+// Apify SDK - toolkit for building Apify Actors (Read more at https://docs.apify.com/sdk/js/).
 import { Actor } from 'apify';
 
+// The init() call configures the Actor for its environment. It's recommended to start every Actor with an init().
```

Comment:
Not sure now, do we have some section explaining why those methods are needed in more detail? 

This `init` call literally switches the underlying storage clients - when you are on platform, it will enable the apify client instead of the default memory storage. Without this, the code on the platform will remain using the memory storage, so it won't save anything to the apify dataset, nor will it respect the input from KVS.

Reply:
Same here - not related to the PR. 

But good question! I was actually looking for this some time ago and the best answer was coming from Python docs e.g.:

<img width="871" alt="Snímek obrazovky 2023-08-08 v 16 31 34" src="https://github.com/apify/apify-docs/assets/45016873/f00fb324-2855-462f-b9e2-9b733b2f7e1e">

I'm actually not sure if there is anything better. Maybe @mtrunkat would know?

Reply:
We have it described in the crawlee docs here

https://crawlee.dev/docs/guides/apify-platform#running-crawlee-code-as-an-actor

Reply:
Yeah, but we should probably have it in Apify docs too, right? 😄

Reply:
Yes, that's exactly my point.

Reply:
I created a ticket here https://github.com/apify/apify-docs/issues/670